### DISCUSSION TITLE
Project set up help

### DISCUSSION THREAD

**Discussion by Simplyalex99** [heybereket/oasis#217] [Q&A]
Hello, I read over the instructions  on the wikie API and Web link.  However, I am facing issues for the API set up, for step 3 the ormconfig.ts file what would be the database name? And in addition, could someone please walk me through the steps to set up the API, as I tried following the outline exactly like. creating the .env files with github tokens and installing required packages and redis but facing lots of build issues. Thanks in advance!

**Comment by Simplyalex99** [comment]
^ Also, next js isn't supported on my mac due to my node version, I do plan to install node again and try installing next js, but is there version for react js since react works for me, or is that no longer available? Thanks

**Reply by HenryLeC** [reply]
Hello, we can provide better help if you join our discord https://discord.gg/cr4r3Ent 😄

**Reply by jzeuzs** [reply]
😀😀😀😀😀😀😀😀😀😀😀😀😀😀😀😀😀

**Comment by HenryLeC** [comment]
Your ormconfig.ts file should look something like this
```ts
// Copy this file into "ormconfig.ts" in this same directory
// You can change the TypeORM connection options here
import { ConnectionOptions } from 'typeorm';
import { joinRoot } from '@utils/common/rootPath';

/// SQLite (simple flat file storage intended for development instances)
export const ormconfig: ConnectionOptions = {
  type: 'sqlite',
  /* Be sure to change this to the location you'd like the sqlite file to be saved. */
  database: 'db.sqlite',
  entities: [joinRoot('./entities/*.*')],
  synchronize: true,
};

export default ormconfig;
```

With SQLite the database name is anything you want, it will just make a new file. With PostgreSQL, it requires a little but more setup

**Reply by HenryLeC** [reply]
I also just pushed a new commit that allows you to not supply any OAuth creds as requiring creds from GitHub, Discord, and Twitter was a little difficult. You will still need to add creds to enable auth, but you can now choose which provider
